twpca/model.py

```diff
@@ -196,6 +197,40 @@ def transform(self, X=None):
 
         return self._sess.run(warp.warp(X_tf, self._inv_warp))
 
+    def predict(self, X=None):
+        """Return model prediction of activity on each trial.
+
```

Comment:
can you add docstring for the expected size of X and a check that the size of X matches the temporal factors